css/styles.css

```diff
@@ -0,0 +1,86 @@
+body {
+  width: 100%;
+  background-color: #f0f0f0;
+}
+
+h2 {
+  font-size: 35px;
+  font-family: 'Proza Libre', sans-serif;
+  font-weight: 600;
+  margin: 5px 0;
+}
+
+input {
+  margin-bottom: 15px;
+}
+
+.label {
+  margin-top: 0;
+  font-family: 'Proza Libre', sans-serif;
+}
+
+.note-label {
+  font-family: 'Proza Libre', sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-full {
+  display: flex;
+  flex-direction: column;
```

Comment:
You shouldn't change the behavior of the layout class like this. Make this a separate class so you don't corrupt your full-width column class for future uses.